lib/src/solver/package_lister.dart

```diff
@@ -52,6 +52,11 @@ class PackageLister {
   /// The set of package names that were overridden by the root package.
   final Set<String> _overriddenPackages;
```

Comment:
Do we still need this? Won't it contain all the keys of `_overriddenPackageRanges`?

Reply:
Done